glossary.yml

```
@@ -1749,6 +1749,24 @@
     def: >
       An unambiguous name of the form `package::thing`.
 
+- slug: function
+  ref:
+    - functions
+    - command
+  en:
+    term: "function"
+    def: >
+      A code block which gathers a sequence of operations into a whole, preserving it for ongoing use
+      by defining a set of tasks that takes zero or more required and optional arguments as inputs
+      and return expected outputs (return values), if any. Functions enable repeating these defined
```

```suggestion
      and returns expected outputs (return values), if any. Functions enable repeating these defined
```

I missed an s! Darn collective noun verb conjugations....